pkg/apis/core/types.go

```diff
@@ -405,6 +405,12 @@ type PersistentVolumeClaimSpec struct {
 	// Resources represents the minimum resources required
 	// +optional
 	Resources ResourceRequirements
+	// AllocatedResources represents the resources that was requested to satisfy user
+	// request. This field is not meant to be set by the user but is used for tracking
+	// volume quota.
```

Comment:
Add a comment indicating this field is alpha, similar to:

https://github.com/kubernetes/kubernetes/blob/a26e5881d8aa38aaa0a56c78050fd4ea14611c9f/pkg/apis/core/types.go#L2638

Reply:
fixed